lib/Compilation.js

```diff
@@ -3271,7 +3271,7 @@ Or do you want to use the entrypoints '${name}' and '${runtime}' independently o
 					this.logger.log(
 						`${Math.round(
 							(100 * statModulesGenerated) /
-								(statModulesGenerated + statModulesFromCache)
+								(statModulesGenerated + statModulesFromCache) || 0
```

Comment:
Weird, I think one of variables has invalid value, I think will be great to found it and fix it

Reply:
> Weird, I think one of variables has invalid value, I think will be great to found it and fix it

ok